src/main.rs

```diff
@@ -92,6 +93,27 @@ fn film_id_from_response(response: letterboxd::SearchResponse) -> Vec<String> {
         .collect::<Vec<String>>()
```

Comment:
No need to specify type, which would be inferred from the return type otherwise.